packages/thirdweb/src/utils/random.ts

```diff
@@ -3,8 +3,13 @@ import { uint8ArrayToHex } from "./encoding/hex.js";
 /**
  * @internal
  */
-export async function randomBytes(length = 32) {
-  return uint8ArrayToHex(
-    globalThis.crypto.getRandomValues(new Uint8Array(length)),
-  );
+export function randomBytes(length = 32) {
```

Comment:
probably rename this
```suggestion
export function randomBytesHex(length = 32) {
```